samples/shopping-cart-with-view/README.md

```diff
@@ -0,0 +1,32 @@
+# Shopping Cart Sample (with View)
+This example builds on the previous shopping cart sample by taking a look at the domain model and improving it. This sample
+adds user authentication, more cleanly separates the data types across roles and responsibilities, and adds a view that can be used by consumers to query the read model.
```

Comment:
```suggestion
adds user authentication, more cleanly separates the data types across roles and responsibilities, and adds a view that can be used by users to query the read model.
```
to avoid confusion with Consumer component